nad/val_def.demo

```diff
@@ -0,0 +1,83 @@
+
+-----------------------------------------------------------------------
+
+      INTEGRATING DEFINITION AND VALIDATION OF GEODETIC SYSTEMS
```

Comment:
Perhaps this should be placed in `examples/` instead?

Reply:
Sorry, @kbevers - two comments from you popped up when I pressed "squash and merge". Will handle these in a later PR.

Reply:
And there's a few more coming. Would have been nice to be able to do a final review before merging.